firebase-ml-modeldownloader/ktx/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (C) 2021 The Android Open Source Project
```

Comment:
Use the Google copyright notice, as in 

https://github.com/rosariopfernandes/firebase-android-sdk/blob/master/firebase-datatransport/src/main/AndroidManifest.xml